tables/customizing-the-table-toolbar.html

```diff
@@ -32,26 +30,26 @@
             selector:'#editor',
 
             // Toggle the menubar off if it isn't needed
-            // https://www.tiny.cloud/docs/configure/editor-appearance/#menubar
+            // https://www.tiny.cloud/docs/tinymce/latest/menus-configuration-options/#menubar
             menubar: false,
 
             // Tip! To make TinyMCE leaner, only include the plugins you actually need.
             plugins: 'link lists image table code',
```

Comment:
```suggestion
            plugins: 'link lists image table code emoticons',
```
emoticons is in the toolbar config but missing from here